librz/util/float/float.c

```diff
@@ -224,7 +224,11 @@ RZ_API bool rz_float_set_from_single(RZ_NONNULL RzFloat *f, float value) {
 		return false;
 	}
 
+#if RZ_SYS_ENDIAN
```

Comment:
does this behaviour depend on the build machine or on the binary under analysis?

Reply:
@ret2libc it's used for setting bitvectors from "native" floats, which are machine-dependent, not binary-dependent.

Reply:
these no. the issue is that on BE machines the stack/heap stored values are in BE format so when we convert them to RzFloat they need to be "read" with the correct endianness.
It's kinda like the endianness for `htol`, depending on the arch you have an empty function or a byte conversion function.